SU2_CFD/include/solvers/CEulerSolver.hpp

```diff
@@ -29,7 +29,7 @@
 
 #include "CFVMFlowSolverBase.hpp"
 #include "../variables/CEulerVariable.hpp"
-
+#include "../output/CTurboOutput.hpp"
```

Comment:
This is not a good idea, solvers do not have output classes
```suggestion
#include "../output/CTurboOutput.hpp"

```

Reply:
Thanks for the spot this was left over from some now redundent functions moved to iteration class